data/system-db.edn

```diff
@@ -164,25 +161,6 @@
           #:project{:dir
                     "/opt/scheduling/projects/sur-optical-fiber/db/",
                     :id :sur-optical-fiber,
-                    :name "SUR Optical Fiber"}
```

Comment:
These projects don't exist in the "common" projects (under data/projects). This breaks the app on start-up (No such DB error).